src/ophyd_async/epics/motor.py

```diff
@@ -83,12 +88,16 @@ def __init__(self, prefix: str, name="") -> None:
         self.motor_done_move = epics_signal_r(int, prefix + ".DMOV")
         self.low_limit_travel = epics_signal_rw(float, prefix + ".LLM")
         self.high_limit_travel = epics_signal_rw(float, prefix + ".HLM")
+        self.offset_freeze_switch = epics_signal_rw(OffsetMode, prefix + ".FOFF")
+        self.high_limit_switch = epics_signal_r(float, prefix + ".HLS")
+        self.low_limit_switch = epics_signal_r(float, prefix + ".LLS")
```

Comment:
Does `bool` work? If not it should at least be `int`.
```suggestion
        self.high_limit_switch = epics_signal_r(bool, prefix + ".HLS")
        self.low_limit_switch = epics_signal_r(bool, prefix + ".LLS")
```
I'm confused that this didn't complain when you try to connect to a real motor, it should have errored saying `DBR_SHORT can't be coerced to type float`

Reply:
ah - turns out i was testing against the wrong thing (our motor wrapper) - whoops! 

yes - i think it needs to be `int` as i get: 
```
    high_limit_switch: TypeError: TE:NDW2932:MOT:MTR0101.HLS with inferred datatype int cannot be coerced to bool
    low_limit_switch: TypeError: TE:NDW2932:MOT:MTR0101.LLS with inferred datatype int cannot be coerced to bool
```